tuf/api/serialization/json.py

```diff
@@ -47,19 +49,31 @@ class JSONSerializer(MetadataSerializer):
     Attributes:
         compact: A boolean indicating if the JSON bytes generated in
                 'serialize' should be compact by excluding whitespace.
+        validate: Whether to validate when serializing.
```

Comment:
Can you try to improve this? It's not easy but the idea is to give the reader enough information to choose True or False without reading the source code: could try to mention that it exists to prevent programming errors from leading to changes in metadata serialize/deserialize cycle , but maybe also the cost.